x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -356,7 +356,11 @@ private static RoleDescriptor buildViewerRoleDescriptor() {
                 RoleDescriptor.ApplicationResourcePrivileges.builder()
                     .application("kibana-.kibana")
                     .resources("*")
-                    .privileges("read").build() },
+                    .privileges("read").build(),
+                RoleDescriptor.ApplicationResourcePrivileges.builder()
+                    .application("kibana-.*")
```

Comment:
```suggestion
                    .application("kibana-*")
```

I think this is the right syntax, given the comment in https://github.com/elastic/elasticsearch/pull/71710#discussion_r614007999 and a chat I had with @legrego 

I'll leave to Larry to comment futher.

Reply:
You're probably right, let's wait for @legrego to verify

Reply:
@jkakavas, @bytebilly's suggestion is the correct syntax: There should not be a `.` in the application name for "reserved" privileges